src/lib/web/templates/home/index.hbs

```diff
@@ -1,5 +1,59 @@
-<div class="margin-b">
-  <a href="/home/breweries">
-    <button class="btn btn-outline-secondary">Breweries</button>
-  </a>
+<script>
+  window.yardstick = window.yardstick || {}
+  window.yardstick.breweries = {{{breweriesJson}}}
+
+  {{!--TODO: get current user location--}}
+
+  function initMap() {
```

Comment:
Maybe `initMap()` should be in a separate `.js` file and reused in `home/show.hbs`?

Reply:
Handlebars won't work with script tags with `src=""` attribute. It requires javascript to be passed as json from the node and inserted as `{{foobar}}` within the script tags. I failed to make that work. 
Escaping, like in 
`<script src="main.js"><{{!}}/script>`
is another solution and seems much simpler but it did not work as well.